website/docs/api/top-level.md

```diff
@@ -158,6 +165,13 @@ if no GPU is available. If data has already been allocated on CPU, it will not
 be moved. Ideally, this function should be called right after importing spaCy
 and _before_ loading any pipelines.
 
+<Infobox variant="warning" title="Jupyter notebook usage">
```

Comment:
Maybe we should also link to the `ipython` issue here as well (e.g. as "see this IPython issue for background on this")? If a user comes across this later and the issue happens to already be resolved, it'll be more clear why the warning is there and that it's not a general limitation of spaCy.

Reply:
Maybe link to the new v3 usage section that does have the link?

Reply:
Yeah, or that 👍